STYLEGUIDE.md

```diff
@@ -0,0 +1,52 @@
+# Shopware Development Documentation Style Guide
```

Comment:
Can this file be removed? Seems to be duplicated by `source/contribute/index.md`